client/a8c-for-agencies/sections/overview/sidebar/index.tsx

```diff
@@ -0,0 +1,17 @@
+import OverviewSidebarHelp from './help';
+import OverviewSidebarQuickLinks from './quick-links';
+import OverviewSidebarResources from './resources';
+import OverviewSidebarVideo from './video';
+
+const OverviewSidebar = () => {
+	return (
+		<div className="overview-sidebar">
+			<OverviewSidebarVideo />
+			<OverviewSidebarQuickLinks />
+			<OverviewSidebarResources />
+			<OverviewSidebarHelp />
```

Comment:
I think we can remove `OverviewSidebarVideo,` `OverviewSidebarResources` & `OverviewSidebarHelp` for now as we will not implement them for the MVP. You can refer to the Technical Scoping P2 for more info.

Reply:
Yes, thanks it makes sense